### PR TITLE
Feature/add page tabs to other pages

### PR DESCRIPTION
**Pull request by minevala**
## Description
- Use PageTabs also in other tabs-pages
- Fix sidenavigation active link resolving
 
## Motivation and Context
- PageTabs add URL-support to tabs and deeplinking

## How Has This Been Tested?
- Locally on dev machine

👉 [Demo](https://city-of-helsinki.github.io/hds-demo/docs-pagetabs-other-pages/)

### CODE REVIEW

**Review by ronijaakkola — APPROVED** [review]

File: new-site/src/components/layout.js [hunk 1]
```diff
@@ -86,6 +86,18 @@ const resolveNavigationLinkByPathAndLevel = (parentPath, level) => (page) => {
   return pathParts.length === level && pathParts.slice(0, -1).every((pathPart) => parentPath.includes(pathPart));
 };
 const sortByPageTitle = (pageA, pageB) => pageA.title.localeCompare(pageB.title);
+const isMatchingParentLink = (link, slug) => {
+  const slugParts = resolvePathPartsFrom(slug);
+  const linkParts = resolvePathPartsFrom(link);
+  const slugPartsWithoutLast = slugParts.slice(0, -1);
+
+  return (
+    slugPartsWithoutLast.length >= 2 &&
```
Comment (VilleMiekkoja) [inline new-site/src/components/layout.js:95]: We might be able to simplify this like so:

```
slugPartsWithoutLast.length === linkParts.length &&
slugPartsWithoutLast.length >= 2
```

Check equality first, after that we only need to check the length of one (linkParts or slugPartsWithoutLast)

File: new-site/src/docs/elements/design-tokens/colour/index.mdx [hunk 2]
```diff
@@ -1,38 +1,86 @@
...
-    <Usage />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Accessibility />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Tokens />
-  </Tabs.TabPanel>
-</Tabs>
+import { Link } from 'hds-react';
```
Comment (VilleMiekkoja) [inline new-site/src/docs/elements/design-tokens/colour/index.mdx:7]: Do we need this import? Link should be in global space.

File: new-site/src/docs/elements/design-tokens/typography/index.mdx [hunk 3]
```diff
@@ -1,33 +1,81 @@
 ---
 slug: '/elements/design-tokens/typography'
-title: 'Typography'
+title: 'Typography tokens - Usage'
 nav_title: 'Typography'
 ---
 
-import { StatusLabel, Tabs } from 'hds-react';
```
Comment (VilleMiekkoja) [inline new-site/src/docs/elements/design-tokens/typography/index.mdx:7]: No need to import this Link I think

File: new-site/src/docs/elements/design-tokens/typography/index.mdx [hunk 4]
```diff
@@ -1,33 +1,81 @@
...
+import { Link } from 'hds-react';
```
Comment (VilleMiekkoja) [inline new-site/src/docs/elements/design-tokens/typography/index.mdx:7]: No need to import this Link I think

File: new-site/src/components/layout.js (hunk 1 above)
Comment (minevala) [reply new-site/src/components/layout.js:95]: Good catch 👍 
- [ ] fix condition

File: new-site/src/docs/elements/design-tokens/colour/index.mdx (hunk 2 above)
Comment (minevala) [reply new-site/src/docs/elements/design-tokens/colour/index.mdx:7]: - [ ] remove unnecessary import

File: new-site/src/docs/elements/design-tokens/typography/index.mdx (hunk 4 above)
Comment (minevala) [reply new-site/src/docs/elements/design-tokens/typography/index.mdx:7]: - [ ] remove unnecessary import

**Review by VilleMiekkoja — APPROVED** [review]
Nice 👍

**Review by VilleMiekkoja — APPROVED** [review]
Good improvement 👍

**Pull request merged**